tests/integration/base_deploy_test.go

```diff
@@ -34,30 +39,53 @@ var (
 )
 
 //Test to make sure all rook components are installed and Running
-func checkIfRookClusterIsInstalled(k8sh *utils.K8sHelper, s suite.Suite, oNamespace string, cNamespace string) {
-	logger.Infof("Make sure all Pods in Rook Cluster %s are running", cNamespace)
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-operator", oNamespace, 1, "Running"),
+func checkIfRookClusterIsInstalled(s suite.Suite, k8sh *utils.K8sHelper, testClient *clients.TestClient,
+	opNamespace string, clusterNamespace string) {
+
+	logger.Infof("Make sure all Pods in Rook Cluster %s are running", clusterNamespace)
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-operator", opNamespace, 1, "Running"),
 		"Make sure there is 1 rook-operator present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-agent", oNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-agent", opNamespace, 1, "Running"),
 		"Make sure there is 1 rook-agent present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-api", cNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-api", clusterNamespace, 1, "Running"),
 		"Make sure there is 1 rook-api present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mgr", cNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mgr", clusterNamespace, 1, "Running"),
 		"Make sure there is 1 rook-ceph-mgr present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-osd", cNamespace, 1, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-osd", clusterNamespace, 1, "Running"),
 		"Make sure there is at lest 1 rook-ceph-osd present in Running state")
-	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mon", cNamespace, 3, "Running"),
+	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mon", clusterNamespace, 3, "Running"),
 		"Make sure there are 3 rook-ceph-mon present in Running state")
+
+	logger.Infof("Testing cluster %s setup health", clusterNamespace)
```

Comment:
here is where we now check cluster health in addition to the cluster pods running.